fury/__init__.py

```diff
@@ -51,14 +49,13 @@ def get_info(verbose=False):
         "sys_platform": sys.platform,
         "numpy_version": numpy.__version__,
         "scipy_version": scipy.__version__,
-        # TODO: add pygfx version
+        # TODO: Add pygfx version if applicable
```

Comment:
I don't remember the last decision about pygfx version, so I didn't change this.